lib/modulesync/git.rb

```diff
@@ -45,23 +45,22 @@ def self.switch_branch(repo, branch)
       end
     end
 
-    def self.pull(git_base, name, branch, project_root, opts)
-      puts "Syncing #{name}"
-      Dir.mkdir(project_root) unless Dir.exist?(project_root)
+    def self.pull(puppet_module, branch)
+      puts "Syncing '#{puppet_module.given_name}'"
 
       # Repo needs to be cloned in the cwd
-      if !Dir.exist?("#{project_root}/#{name}") || !Dir.exist?("#{project_root}/#{name}/.git")
+      if !Dir.exist?("#{puppet_module.working_directory}/.git")
```

Comment:
While badly named, it looks like this should be using `module_file()` as well. Not strictly needed now, but a good future improvement.

Reply:
OK for futurs improvement, I'll write it down in my TODO file.